src/types.ts

```diff
@@ -57,12 +57,16 @@ export type PartialDeploymentHistory<T extends DeploymentBase> = {
     offset: number
     limit: number
     moreData: boolean
+    next?: string
+    lastEntityId?: string
```

Comment:
What do you think if we try to use `lastId`, just for consistency with other paginated endpoints (such as wearables)?